src/components/AddOrUpdateCandidate/AddOrUpdateCandidate.tsx

```
@@ -113,32 +116,51 @@ export const AddOrUpdateCandidate: VFC<AddOrUpdateCandidateProps> = (props) => {
     const onSubmit: SubmitHandler<FormValues> = variant === 'new' ? create : update;
     const onOutstaffVendorIdChange = (outstaffVendorId: string) => setValue('outstaffVendorId', outstaffVendorId);
 
-    const { ref: refName, ...restName } = register('name', validationRules.nonEmptyString);
-    const { ref: refEmail, ...restEmail } = register('email', {
-        required: false,
-    });
-
     return (
-        <FormContainer
-            submitButtonText={variant === 'new' ? tr('Add candidate') : tr('Save candidate')}
-            onSubmitButton={handleSubmit(onSubmit)}
-            submitButtonDisabled={isSubmitting}
-        >
-            <FormInput label={tr('Full name')} helperText={errors.name?.message} forwardRef={refName} {...restName} />
-            <Select
-                options={options}
-                value={watch('outstaffVendorId')}
-                onChange={onOutstaffVendorIdChange}
-                text={tr('Employment')}
-            />
-            <FormInput label={tr('Email')} helperText={errors.email?.message} forwardRef={refEmail} {...restEmail} />
-            <FormPhoneInput
-                name="phone"
-                control={control}
-                label={tr('Phone number')}
-                defaultValue={candidate?.phone || ''}
-                options={{ required: false }}
-            />
-        </FormContainer>
+        <StyledFormCard>
+            <Form onSubmit={handleSubmit(onSubmit)}>
+                <Fieldset>
+                    <FormInput
+                        {...register('name')}
+                        label={tr('Full name')}
+                        autoComplete="off"
+                        flat="bottom"
+                        error={isSubmitted ? errors.name : undefined}
```

Why hide the error under the ternary?

---

I took it from issues:/